docs/Integrating Your HMI/Vehicle Data/index.md

```diff
@@ -0,0 +1,169 @@
+|electronicParkBrakeStatus|[Common.ElectronicParkBrakeStatus](https://smartdevicelink.com/en/guides/hmi/common/enums/#electronicparkbrakestatus)|The status of the park brake as provided by Electric Park Brake (EPB) system|
+|engineOilLife|Float|The estimated percentage of remaining oil life of the engine|
+|engineTorque|Float|Torque value for the engine (in N*m) on non-diesel variants|
+|externalTemperature|Float|The external temperature in degrees celsius|
+|fuelLevel|Float|The fuel level in the tank (as a percentage value)|
+|fuelLevel_State|[Common.ComponentVolumeStatus](https://smartdevicelink.com/en/guides/hmi/common/enums/#componentvolumestatus)|The status value corresponding to the general fuel level in the tank|
+|fuelRange|[Common.FuelRange](https://smartdevicelink.com/en/guides/hmi/common/structs/#fuelrange)|The estimate range in KM the vehicle can travel based on fuel level and consumption. Contains information on all fuel sources available to the vehicle (eg. GASOLINE and BATTERY for hybrid vehicles).|
+|gps|[Common.GPSData](https://smartdevicelink.com/en/guides/hmi/common/structs/#gpsdata)|Location data from the onboard GPS in the vehicle|
+|headLampStatus|[Common.HeadLampStatus](https://smartdevicelink.com/en/guides/hmi/common/structs/#headlampstatus)|The current status of each of the head lamps|
+|instantFuelConsumption|Float|The instantaneous fuel consumption of the vehicle in microlitres|
+|odometer|Integer|The odometer value in kilometers|
+|prndl|[Common.PRNDL](https://smartdevicelink.com/en/guides/hmi/common/enums/#prndl)|The current status of the gear shifter|
+|rpm|Integer|The number of revolutions per minute of the engine|
+|speed|Float|The vehicle speed in kilometers per hour|
+|steeringWheelAngle|Float|The current angle of the steering wheel (in degrees)|
+|tirePressure|[Common.TireStatus](https://smartdevicelink.com/en/guides/hmi/common/structs/#tirestatus)|Status information for each of the vehicle's tires|
+|turnSignal|[Common.TurnSignal](https://smartdevicelink.com/en/guides/hmi/common/enums/#turnsignal)|The current state of the turn signal indicator|
+|vin|String|Vehicle identification number|
+|wiperStatus|[Common.WiperStatus](https://smartdevicelink.com/en/guides/hmi/common/enums/#wiperstatus)|The current status of the wipers|
+
+## Custom Vehicle Data Items
+
+Starting with [SDL Core version 6.0.0](https://github.com/smartdevicelink/sdl_core/releases/tag/6.0.0), custom vehicle data items can be defined via the policy table. These items are structured in a similar manner to the [MOBILE API](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml) and contained in the `vehicle_data` section of the policy table. For example:
+
+```
+"vehicle_data": {
+    "schema_version": "1.0.0",
+    "schema_items": [
+        ...
+        {
+            "name": "customString",
```

Comment:
The [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0173-Read-Generic-Network-Signal-data.md) has good definitions of each of these properties. Might be worth including in this guide.